definition.tex

```diff
@@ -23,7 +23,7 @@ \section{Defining Proof-of-Burn}
 
 With foresight, we remark that the implementation of $\GenBurnAddress$ and $\BurnVerify$ will typically be deterministic, which alleviates the need for a probabilistic correctness definition.
 
-Naturally, for $\GenBurnAddress$ to generate addresses that ``look'' valid but are unspendable according to the source blockchain protocol requires that the burn protocol respects the format of the source blockchain protocol. We abstract the functionalities of the source blockchain protocol into a \emph{blockchain address protocol}:
+Naturally, for $\GenBurnAddress$ to generate addresses that ``look'' valid but are unspendable according to the blockchain protocol requires that the burn protocol respects the format of the blockchain protocol. We abstract the functionalities of the blockchain protocol into a \emph{blockchain address protocol}:
```

Comment:
Shorten:
* respects the format of the blockchain protocol -> respects its format
* the functionalities of the blockchain protocol -> these functionalities